src/odemis/driver/test/pmtctrl_test.py

```diff
@@ -47,6 +47,20 @@
 CLASS_CTRL = pmtctrl.PMTControl
 KWARGS_CTRL = KWARGS
 
+# TR Control (photon signal detector)
```

Comment:
Please, instead of "TR" use "Time-correlator" or TC. That's a little bit more correct.